app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -15,7 +15,7 @@
       </tr>
       <tr>
         <td>{{t 'Sales (amount)'}}</td>
-        <td>{{currency-symbol this.data.event.paymentCurrency}} {{format-money this.sales}}</td>
+        <td>{{#if this.sales}}{{currency-symbol this.data.event.paymentCurrency}}{{/if}} {{format-money this.sales}}</td>
```

Comment:
Duplicate logic. There should be a single if statement in the entire PR

Reply:
You mean that like I have used if statement in different files, so there should be only one if or condition statement that should handle this?